### ISSUE TITLE
Poor performance with provided pretrained checkpoint

### ISSUE THREAD

**Issue by WXK610** [AVC2-UESTC/OV2VSS#2]
Hello @linxinhao,

When testing with the provided `100000.pth` checkpoint , the mIOU is only 0.006. After retraining the model , the training loss decreased from 3.8 to 1.5, and the train  accuracy  reached  about 45%, but the validation mIOU is still low.

Steps to Reproduce:
First, I followed the Readme to complete the data preprocessing. Then I modified the paths for the data, as well as the paths for the train_class.json, test_class.json and the tempfile. No other modifications were made.

Could you please help verify if this is a known issue? Any guidance on debugging directions would be greatly appreciated. Thank you for your work on this project!

**Comment by bingnoi** [comment]
sorry for the late reply, i have fixed the bug.If the problem still persists, please contact me.
